### ISSUE TITLE
Icon on taskbar notification area

### ISSUE THREAD

**Issue by LuigiImVector** [stnkl/EverythingToolbar#145]
I've noticed that if there are too few icons in the taskbar notification area not only the logo is shown but also everything else, I find it a bit annoying, it would be convenient if there was an option that always showed the contracted or extended version.
![2021-04-15 19-58-26](https://user-images.githubusercontent.com/59611597/114917901-c4a49b80-9e26-11eb-8be6-a0024cede462.gif)

**Comment by Horst12** [comment]
You have to unlock the taskbar, change the siize of the Toolbar icon
and lock the taskbar gain.
Then ist always shown as wanted.
![Screenshot - 15 04 2021 , 21_54_10](https://user-images.githubusercontent.com/38587340/114930332-279d2f00-9e35-11eb-9486-e10a2710fd11.png)

**Comment by LuigiImVector** [comment]
As I showed in the gif I unlocked the taskbar and changed the size of the Toolbar icon but as soon as I lock the taskbar again it comes back with "Search", I just want the icon.

**Comment by CoenraadS** [comment]
I agree, on my setup I only have the small icon on the right side of my taskbar. What I expected was the extended version on the left of my taskbar like in the screenshot in the readme.

**Comment by KirkH420** [comment]
I had the same problem.
You need to unlock the taskbar
Drag the handlebar to the right of the Everything Icon.
  -Drag this ALL THE WAY over to your start button.
  -It will look weird at first
Next grab the other handlebar that's to the left of the Everything Icon
  -Drag it over towards your start button
  -Stop dragging when the search bar is the size that you want it.
Lock the task bar.

In your gif clip, the reason that it's closing like that is because, I think it needs to be sized by stretching from the right side of the toolbar, which you can do once you move it over by Start.
